resources/puppetlabs/lein-ezbake/template/global/tasks/build.rake

```diff
@@ -58,7 +62,8 @@ namespace :pl do
         if Pkg::Config.build_pe
```

Comment:
My standard gripe. I would rather see it this way to create a sense of normal/exceptional case.

```ruby
platform_path = "pe/deb/#{platform}"
if !Pkg::Config.build_pe
  platform_path = "deb/#{platform}/#{repo}".sub(/\/$/, '')
end
```